### DISCUSSION TITLE
Non-blinking cursor?

### DISCUSSION THREAD

**Discussion by emendelson** [dosemu2/dosemu2#1906] [Ideas]
Is it possible to make the cursor NOT blink, as in DOSBox-X and vDos? If there's an option that does this, I've missed it. The blink_rate option doesn't seem to let me turn the blinking off. I've tried installing the old NoBlink utility, but it slows down the cursor to the point where the DIR command (for example) is unusable:

http://www.columbia.edu/~em36/noblink.html

It would be nice to have a non-stressful non-blinking cursor in dosemu2, but I know I'm asking a lot.

**Comment by stsp** [comment]
Fixed for `$_X_blinkrate = (0)`

**Reply by emendelson** [reply]
Perfect. I'll look forward to the updated version. (As you saw, 0 produced very fast blinking, not steady-state.)
